File: backend/src/main/java/pl/maciej/matyjasek/voting/app/voter/dto/VoterDto.java
package pl.maciej.matyjasek.voting.app.voter.dto;

import lombok.*;

import java.util.UUID;

@Value
public class VoterDto {

	UUID uuid;
	@NonNull
	String firstName;
	@NonNull
	String lastName;
	@NonNull
	Boolean hasVoted;
}

File: frontend/voting/src/app/vote/voter/voterDto.ts
export class VoterDto {
  uuid: string;
  firstName: string;
  lastName: string;
  hasVoted: boolean;


  constructor(uuid: string, firstName: string, lastName: string, hasVoted: boolean) {
    this.uuid = uuid;
    this.firstName = firstName;
    this.lastName = lastName;
    this.hasVoted = hasVoted;
  }
}
